cvat/schema.yml

```diff
@@ -1249,6 +1239,83 @@ paths:
               schema:
                 $ref: '#/components/schemas/ClientEvents'
           description: ''
+  /api/events/file/export:
```

Comment:
IMO, `/file/` is so generic, it's meaningless. Since there's only one thing that can be exported WRT events, I would suggest to just put this at `/api/events/export`.